src/main/java/org/geysermc/discordbot/commands/moderation/ForumPostCommand.java

```diff
@@ -0,0 +1,368 @@
+                .queue();
+        }
+    }
+
+    public static class CloseOldPostSubCommand extends SlashCommand {
+        public CloseOldPostSubCommand() {
+            this.name = "close-old";
+            this.help = "Close old posts";
+            this.userPermissions = new Permission[] { Permission.MESSAGE_MANAGE };
+            this.options = Collections.singletonList(new OptionData(OptionType.INTEGER, "days", "The minimum age in days of posts that will be closed in bulk", true));
+        }
+
+        @Override
+        protected void execute(@NotNull SlashCommandEvent event) {
+            int days = Objects.requireNonNull(event.getOption("days")).getAsInt();
+            Checks.notNull(event.getGuild(), "server");
+
+            ForumChannel forumChannel = ServerSettings.getForumChannel(event.getGuild());
+            if (forumChannel == null) {
+                event.reply("Forum channel not found.").queue();
+                return;
+            }
+
+            for (ThreadChannel channel : forumChannel.getThreadChannels()) {
+                if (channel.isArchived() || channel.isLocked() || channel.isPinned()) {
+                    continue;
+                }
+                OffsetDateTime channelCreated = channel.getTimeCreated();
+                OffsetDateTime closeTime = OffsetDateTime.now().minusDays(days);
+                if (channelCreated.isBefore(closeTime)) {
+                    channel.getManager().setArchived(true).queue();
+                }
+            }
+
+            event.reply("All forum posts older than " + days + " days have been closed!").queue();
+        }
+    }
+
+    private static List<ForumTag> potentialTags(List<ForumTag> tags, String query) {
+        List<ForumTag> potential = new ArrayList<>();
+
+        // Search the providers by an exact starting match
+        for (ForumTag tag : tags) {
+            if (tag.getName().toLowerCase().startsWith(query.toLowerCase())) {
+                potential.add(tag);
+            }
+        }
+
+        // Find the best similarity
+        for (ForumTag tag : tags) {
+            double similar = DicesCoefficient.diceCoefficientOptimized(query.toLowerCase(), tag.getName().toLowerCase());
+            if (similar > 0.2d) {
+                potential.add(tag);
+            }
+        }
+
+        // Send a message if we don't know what provider
+        return potential;
+    }
+
+    private static boolean isForumChannel(@NotNull SlashCommandEvent event) {
+        Guild server = event.getGuild();
+        if (server == null) return false;
+        ForumChannel forumChannel = ServerSettings.getForumChannel(server);
+        if (forumChannel == null) return false;
+        if (event.getChannel().asThreadChannel().getParentChannel().getId().equals(forumChannel.getId())) {
+            return true;
+        }
+        event.reply("Command can only be used in forum channels").queue();
+        return false;
```

Comment:
```suggestion
        return false;
```
All usages of this method already send a message. (If this was kept the method name would need a change as well)